package.json

```diff
@@ -112,7 +112,6 @@
     "lint:sol": "solhint 'example/**/contracts/*.sol' 'truffle/contracts/*.sol'"
   },
   "imports": {
```

Comment:
Let's remove or clean up the entire `imports` section in order to avoid using or adding other aliases declared here unless we find a solution to this issue.